src/test/java/hw2/ex1/HomePageDataProvider.java

```diff
@@ -0,0 +1,42 @@
+package hw2.ex1;
+
+import org.testng.annotations.DataProvider;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class HomePageDataProvider {
+    public static Map<String, String> benefitIcons = new HashMap<>();
+
+    static {
+        benefitIcons.put("icons-benefit icon-practise", "To include good practices\n" +
+                "and ideas from successful\n" +
+                "EPAM project");
+        benefitIcons.put("icons-benefit icon-custom", "To be flexible and\n" +
+                "customizable");
+        benefitIcons.put("icons-benefit icon-multi", "To be multiplatform");
+        benefitIcons.put("icons-benefit icon-base", "Already have good base\n" +
+                "(about 20 internal and\n" +
+                "some external projects),\n" +
+                "wish to get more…");
+    }
+
+    @DataProvider(name = "MenuElements")
+    public static Object[][] menuElements() {
```

Comment:
What benefit here from using `DataProvider`? Why is it static?

Reply:
I decided to export some long text expected data from testscript. And as we are working with TestNG I thought it would be better to use its tools instead of fields. 
It is static because it was autogenerated (alt+Ins) and I didn't checked.